assets/src/styles/blocks/OldENForm/components/_enform.scss

```diff
@@ -171,7 +173,6 @@
 
       p {
         font-size: .75rem;
-        font-family: $roboto !important;
```

Comment:
The `!important` here was not needed after all? 🤔

Reply:
I think I tested if it did something on the live content before removing it, but let me re-check :eyes:

Reply:
It already uses Roboto because of https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/902e5e4322269414a2def75e6399cd679168b61a/assets/src/styles/blocks/OldENForm/components/_enform.scss#L101-L103